communities/src/lib.rs

```diff
@@ -157,17 +157,27 @@ decl_event!(
     {
         /// A new community was registered \[who, community_identifier\]
         CommunityRegistered(AccountId, CommunityIdentifier),
-        /// CommunityMetadata was updated \[who, community_identifier\]
+        /// CommunityMetadata was updated \[community_identifier\]
         MetadataUpdated(CommunityIdentifier),
-        /// A community's nominal income was updated \[who, community_identifier\]
-        NominalIncomeUpdated(CommunityIdentifier),
-        /// A community's demurrage was updated \[who, community_identifier\]
-        DemurrageUpdated(CommunityIdentifier),
+        /// A community's nominal income was updated \[community_identifier, new_income\]
```

Comment:
Here I had to make a small workaround due to `substrate-fixed` not implementing some traits. The value is displayed as the result from `nominal_income.to_bits()`.

Reply:
ok. what traits?

Reply:
This turned out to be an error on my side.

I put `NominalIncome` (storage) instead of `NominalIncometType` in the event, which gave the trait errors. Putting the correct type in the event solved the issue.